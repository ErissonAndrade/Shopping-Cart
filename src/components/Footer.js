import logo from '../assets/images/logo.png';

const Footer = () => {
    return (
        <footer>
            <img src={logo} alt="company-logo"></img>
        </footer>
    );
};

export default Footer;