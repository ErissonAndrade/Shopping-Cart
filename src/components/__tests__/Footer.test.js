import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

it('renders the logo correctly', () => {
    render(<Footer />);
    const logoImg = screen.getByRole('img');
    expect(logoImg).toHaveAttribute('src', 'logo.png');
    expect(logoImg).toHaveAttribute('alt', 'company-logo');
});

it('renders correctly', () => {
    render(<Footer />);
    const footer = screen.getByRole('contentinfo');
    expect(footer).toMatchSnapshot();
});

