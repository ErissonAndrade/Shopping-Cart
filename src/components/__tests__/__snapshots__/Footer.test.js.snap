// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  <img
    alt="company-logo"
    src="logo.png"
  />
</footer>
`;
